ssl/t1_enc.c

```diff
@@ -157,12 +157,12 @@ int tls1_change_cipher_state(SSL *s, int which)
     struct tls_enable crypto_info;
```

Comment:
We could perhaps remove some #ifdef's here and in tls13_enc.c if we had 'ktls.h' define some typedef like `ktls_crypto_info_t` or the like that was used to declare `crypto_info`.

Reply:
That would be nice

Reply:
Done